### PR TITLE
Added week1 homework

### PR DESCRIPTION
**Pull request by NayanaKamtekar**
I have added week1 homework. Please have a look

### CODE REVIEW

File: week1/week1-homework.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+my_name = 'Nayana'
+print(my_name)
+
+my_age = 34
+print(my_age)
+
+my_info = {'name': 'Nayana', 'age': 34}
+
+print(f"My name is {my_info['name']}. I am {my_info['age']} years old")
```
Comment (Rassibassi) [inline week1/week1-homework.py:13]: Good job!

Just for completeness, f-strings, also work with variables, not only dictionaries.
So the following code is also valid:
```python
print(f"My name is {my_name}. I am {my_age} years old")
```

File: week1/week1-homework.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+# ex:2 Create a list of your 5 favorite movies and store it in the variable
+
+My_fav_movies = ['Shrek 2', 'Frozen', 'Shaun The Sheep', "Howl's Moving Castle", 'Spider-Man: Into the Spider-Verse']
```
Comment (Rassibassi) [inline week1/week1-homework.py:17]: Please stick to `snake_case` or `camelCase`, but `My_fav_movies` is neither :)

File: week1/week1-homework.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+print(sample_dictionary)
+
+
+# ex:5 Write a Python program to calculate body mass index.
+
+def calculateBMI(weight, height):
+    return weight / height ** 2
+
+
+print(f"BMI is {calculateBMI(65, 1.56)}")
```
Comment (Rassibassi) [inline week1/week1-homework.py:38]: Good Job!

File: week1/week1-homework.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+# a=list(set(a))
+# print(f"list with unique values{b}")
+
+dedup = []
+
+for elem in a:
+    if elem not in dedup:
+        dedup.append(elem)
+
+print(dedup)
```
Comment (Rassibassi) [inline week1/week1-homework.py:65]: Good job!
I would use `set` as you did in the commented version :)

File: week1/week1-homework.py [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+
+# ex:10 Accept a string from user and remove the characters which have odd index values of a given string and print them.
+def user(input_string):
+    return ''.join(char for index, char in enumerate(input_string) if index % 2 != 0)
```
Comment (Rassibassi) [inline week1/week1-homework.py:70]: Very nice!

File: week1/week1-homework.py [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+    while (True):
+        if random_num == guess:
+            print("Well guessed!")
+            break
+        else:
+            guess = int(input("Not quite right, guess again: "))
+
+
+#number_guess_game()
```
Comment (Rassibassi) [inline week1/week1-homework.py:91]: Good Job here :)

**Review by Rassibassi — COMMENTED** [review]
Good job! See inline comments

File: week1/week1-homework.py (hunk 4 above)
Comment (NayanaKamtekar) [reply week1/week1-homework.py:65]: `set` doesn't preserve the order of elements from the original list therefore I coded the way I did.

Comment (Rassibassi) [reply week1/week1-homework.py:65]: Ok

**Pull request merged**